examples/calculator/pom.xml

```diff
@@ -6,7 +6,7 @@
     <description>A simple calculator application</description>
     <groupId>com.spotify</groupId>
     <artifactId>calculator-service</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Fair enough :) as long as they have artifact ids that make it clear that they are not anything interesting to depend on.